### PR TITLE
Palí

### PR DESCRIPTION
**Pull request by javirg**

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -30760,7 +30760,9 @@
     "tags": {
       "brand": "Palí",
       "name": "Palí",
-      "shop": "supermarket"
+      "shop": "supermarket",
+      "brand:wikidata": "Q1064887",
+      "brand:wikipedia": "en:Walmart_de_México_y_Centroamérica"
```
Comment (bhousel) [inline config/canonical.json]: remove the `_` - thanks!
```suggestion
      "brand:wikipedia": "es:Walmart de México y Centroamérica"
```

File: config/canonical.json [hunk 2]
```diff
@@ -31356,7 +31358,7 @@
     "tags": {
       "brand": "Tesco",
       "name": "Tesco",
-      "shop": "supermarket"
+      "shop": "supermarket",
```
Comment (bhousel) [inline config/canonical.json]: no trailing `,` here
```suggestion
      "shop": "supermarket"
```

File: config/canonical.json (hunk 1 above)
Comment (bhousel) [reply config/canonical.json]: Also, I don't see any mention of the "Palí" brand on the `en:` page, so I changed it to the `es:` page which mentions it.

File: config/canonical.json (hunk 2 above)
Comment (good-praxis) [reply config/canonical.json]: :thinking: That's odd, this is the trailing comma from master fixed in #1914, how was it committed here?

Comment (bhousel) [reply config/canonical.json]: Yeah I was wondering that too...
The merges are coming fast and furious today, so I'm not surprised that there is a little bit of breakage and that every time we `npm run build` things get a little shuffled around.

**Pull request merged**